src/Controllers/DatabaseController.js

```diff
@@ -62,7 +62,7 @@ const validateQuery = (query: any): void => {
   }
 
   if (query.$or) {
-    if (query.$or instanceof Array) {
```

Comment:
Revert these changes to `$or` and `$and` as are they are causing tests to fail.

They also have nothing to do with the PR.